airflow/config_templates/airflow_local_settings.py

```diff
@@ -78,11 +78,20 @@
         # },
     },
     'loggers': {
-        'airflow.processor' : {
-            'handlers': ['file.processor'],
+        '': {
```

Comment:
ah nice, looks like this will fix the issue for log names that don't start with `airflow`

Reply:
It should indeed